validation/jinja2/validation/_segment_card.html

```diff
@@ -33,7 +33,13 @@
                 {% if auth %}
                     <button type="button" class="button button--fail button--flag right" data-toggle="modal" data-target="#{{ phrase.id }}-modal" data-cy="flag-button">Something's wrong with this entry</button>
                 {% endif %}
-                <p class="card__title--cree" data-cy="transcription">{{ phrase.transcription }}</p>
+                <p class="card__title--cree" data-cy="transcription">
+                    {% if request.COOKIES['macron'] == 'on' %}
```

Comment:
If ever you need to support multiple orthographies, I'd take a look at how itwêwina handles this. Ultimately, it's using a cookie to control the orthography, but it has explicit support for multiple orthographies and how to convert to each one.

See: https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/master/CreeDictionary/morphodict/orthography.py